Dockerfile.e2e-tests

```diff
@@ -0,0 +1,20 @@
+# By default we pin to amd64 sha. Use make docker to automatically adjust for arm64 versions.
```

Comment:
We need to update this comment, i think:
```suggestion
# By default we pin to amd64 sha. Use make docker-e2e to automatically adjust for arm64 versions.
```

Reply:
Gotcha! will update this